docker-storage-setup.sh

```diff
@@ -55,13 +55,13 @@ set -e
 # Note: lvm2 version should be same or higher than lvm2-2.02.112 for lvm
 #       thin pool functionality to work properly.
 POOL_LV_NAME="docker-pool"
-DOCKER_ROOT_LV_NAME="docker-root-lv"
-DOCKER_ROOT_DIR="/var/lib/docker"
+ROOT_LV_NAME="docker-root-lv"
+ROOT_DIR="/var/lib/docker"
 
-DOCKER_STORAGE="/etc/sysconfig/docker-storage"
+STORAGE="/etc/sysconfig/docker-storage"
 STORAGE_DRIVERS="devicemapper overlay overlay2"
 
-DOCKER_METADATA_DIR="/var/lib/docker"
+METADATA_DIR="/var/lib/docker"
```

Comment:
No because we are going to want to pass in the METADATA_DIR via config.  /var/lib/containers will be the METADATA_DIR for OCID.